### PR TITLE
Accessibility Guide [first version]

### PR DESCRIPTION
**Pull request by xtine**
Making a PR of the contents of the collaborative Accessibility Guide document: https://docs.google.com/document/d/1zMS5-jOer8eigc2XwBbOd73UrCODRZJKxV3lPuHSdz8/ 🔒

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,223 @@
...
+
+This is the primary repository of information and resources surrounding accessibility practices at Truss.
 
 ## Tracking work
+
 Issues and their progress will be captured on the [Github project board](https://github.com/trussworks/accessibility/projects/1).
 
 ## Guild
+
 Discussion and support at #g-accessibility
```
Comment (joshmfranklin) [inline README.md:11]: you can deep link to the Slack group with https://slack.com/app_redirect?channel=g-accessibility

File: README.md [hunk 2]
```diff
@@ -1,8 +1,223 @@
...
+
+### Product
+
+- Product should advocate for accessibility as the way we work from the beginning and not a line item to add at the end
+- Product should understand the work as done by other groups
+- Product should prioritized accessibility needs on the roadmap
+- Product should hold space for iteration when needed on accessibility
+
+
+### Useful tools
```
Comment (joshmfranklin) [inline README.md:218]: Maybe for a second pass, but I think we could add a short descriptor about what each of these tools do.

File: README.md (hunk 1 above)
Comment (xtine) [reply README.md:11]: ooh thanks good idea.

File: README.md (hunk 2 above)
Comment (xtine) [reply README.md:218]: Just added descriptions!

**Pull request merged**